examples/cmaccount.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"math/big"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/chain4travel/camino-messenger-contracts/go/contracts/bookingtoken"
+	"github.com/chain4travel/camino-messenger-contracts/go/contracts/cmaccount"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+func main() {
+
+	// Set your CM Account address here
+	cmAccountAddr := common.HexToAddress("<CM_ACCOUNT_ADDRESS>")
+
+	reservedFor := common.HexToAddress("<CM_ACCOUNT_ADDRESS>")
+	uri := "TOKEN_URI"
+	expirationTimestamp := big.NewInt(time.Now().Add(time.Hour).Unix())
+	price := &big.Int{}
+
+	// Set BookingToken address here
+	//bookingTokenAddr := common.HexToAddress("0xe55E387F5474a012D1b048155E25ea78C7DBfBBC")
```

Comment:
Do we need this example? Because we already have one named `booking/mintnbuy.go`, that implements both mint and buy operations with the booking service from the booking pkg.

Reply:
removed